package.json

```diff
@@ -27,64 +27,63 @@
     "@bcrs-shared-components/enums": "1.0.19",
     "@bcrs-shared-components/help-business-number": "1.0.0",
     "@bcrs-shared-components/interfaces": "1.0.39",
-    "@bcrs-shared-components/mixins": "1.0.6",
     "@bcrs-shared-components/nature-of-business": "1.0.3",
     "@bcrs-shared-components/staff-comments": "1.2.0",
     "@bcrs-shared-components/staff-payment": "2.0.1",
     "@bcrs-shared-components/web-chat": "2.0.0",
```

Comment:
I still have to update all these shared components.

Reply:
I tested:
- [ ] breadcrumb
- [ ] business-lookup
- [ ] certify
- [ ] completing-party
- [ ] confirm-dialog
- [ ] contact-info
- [ ] corp-type-module
- [ ] court-order-poa
- [ ] date-picker
- [ ] document-delivery
- [ ] effective-date-time
- [ ] enums
- [ ] help-business-number
- [ ] interfaces
- [ ] nature-of-business
- [ ] staff-comments
- [ ] staff-payment
- [ ] web-chat